modules/nw-udn-cr.adoc

```diff
@@ -6,7 +6,7 @@
 [id="nw-udn-cr_{context}"]
 = Creating a UserDefinedNetwork custom resource
 
-The following procedure sets up a user-defined network that is namespace scoped.
+The following procedure creates a user-defined network that is namespace scoped. Based upon your use case, create your request using either the "my-layer-two-udn.yaml" example for a `Layer2` topology type or the "my-layer-three-udn.yaml" example for a `Layer3` topology type.
```

Comment:
Are these intentionally quotes instead of backticks?
```suggestion
The following procedure creates a user-defined network that is namespace scoped. Based upon your use case, create your request using either the `my-layer-two-udn.yaml` example for a `Layer2` topology type or the `my-layer-three-udn.yaml` example for a `Layer3` topology type.
```

Reply:
Yeah, they were. I wasn't sure tbh how these should be cited. I see that if you were citing a section you'd use " " but then I see your point [here](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections:~:text=or%20example%20names%20for%20individual%20API).